packages/mcp-provider-mobile-web/src/tools/offline-analysis/ruleConfig.ts

```diff
@@ -74,4 +74,77 @@ const noWireConfigReferenceNonLocalPropertyRuleConfig: RuleConfig = {
   config: noWireConfigReferenceNonLocalPropertyRule,
 };
 
-export const ruleConfigs: RuleConfig[] = [noPrivateWireRuleConfig, noWireConfigReferenceNonLocalPropertyRuleConfig];
+// **********  getter related violations **********
+const getterViolation: CodeAnalysisBaseIssueType = {
+  type: 'Violations in Getter',
+  description: 'A getter method does more than just returning a value',
+  intentAnalysis:
+    'The developer attempted to modify component state, access a private property, or reference functions within a getter function.',
+  suggestedAction: dedent`
+        To fix this issue, follow the below instructions:
+        - Eliminate any member variables assignments within getters. Getters should not have side effects. LWC getters should only retrieve data. Never assign values to member variables within a getter. 
+        - Make the properties public by using the @api decorator. If a getter computes a value based on a member variable, make that variable public by adding @api decorator.
+        - Leverage getters for computations. In LWC, getters can return calculated or derived values, not just direct property values.
```

Comment:
I'm not sure I understand this instruction. How does it specifically alleviate the improper access that causes this rule violation?